operator/v1/types_network.go

```diff
@@ -9,6 +9,7 @@ import (
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:path=networks,scope=Cluster
+// +kubebuilder:subresource:status
```

Comment:
What are the implications for the network operator of this change? Have we tested upgrades to make sure this doesn't break?

Reply:
The newly generated CRD was imported and tested in https://github.com/openshift/cluster-network-operator/pull/2506. The upgrade path seems to work fine.

Reply:
Looking at that job, there are three upgrade from 4.17 jobs that all failed indicating CSI issues, are we sure this isn't the cause of that?

Reply:
the status update problems observed in https://github.com/openshift/cluster-network-operator/pull/2506#issuecomment-2383308291 should be fixed with https://github.com/openshift/cluster-network-operator/pull/2506/commits/febd41376cc3553fa47a3be8cb9503337a80934d